app/templates/layouts/_survey.html

```diff
@@ -24,12 +24,18 @@ <h1 class="bar__title">Monthly Business Survey - Retail Sales Index</h1>
 
     {% if data %}
       <div class="info survey-header__info" role="contentinfo">
-        <h2 class="u-vh">Information about this survey</h2>
-        <ul class="info__list">
-          <li class="info__item info__item--half">Survey Code: {{ data.survey_code or '' }}</li>
-          <li class="info__item info__item--half">Period: {{ data.period_str or '' }}</li>
-          <li class="info__item info__item--last">Reference: {{ data.respondent_id or '' }}</li>
-        </ul>
+        <h2 class="info__title">Survey Help</h2>
+        <div class="info__tel">
+          0300 1234 931
+        </div>
+        <dl class="info__list">
+          <dt class="info__dt">ID:</dt>
+          <dd class="info__dd">{{ data.survey_code}}</dd>
+          <dt class="info__dt">Period:</dt>
```

Comment:
You've removed the checks on these data values? This means it'll error if that piece of data is missing. Is that what you wanted to do?